wasp/src/main/java/com/orhanobut/wasp/utils/OkHttpLogInterceptor.java

```diff
@@ -39,7 +42,7 @@ public Response intercept(Chain chain) throws IOException {
         Logger.d("---> END");
 
         long t1 = System.nanoTime();
-        Response response = chain.proceed(request);
+        Response response = chain.proceed(originalRequestBuilder.build());
```

Comment:
I think it would be better to use the copy request object for logging and proceed with the original request object.